MatrixSDK/JSONModels/MXRoomCreateContent.m

```diff
@@ -48,6 +51,12 @@ + (id)modelFromJSON:(NSDictionary *)jsonDictionary
         MXJSONModelSetMXJSONModel(roomCreateContent.roomPredecessorInfo, MXRoomPredecessorInfo, jsonDictionary[kRoomCreateContentPredecessorInfoJSONKey]);
         MXJSONModelSetString(roomCreateContent.roomVersion, jsonDictionary[kRoomCreateContentRoomVersionJSONKey]);
         MXJSONModelSetBoolean(roomCreateContent.isFederated, jsonDictionary[kRoomCreateContentFederateJSONKey])
+        
+        NSString *roomType;
+        
+        MXJSONModelSetString(roomType, jsonDictionary[kRoomCreateContentRoomTypeMSC1772JSONKey]);
+        
+        roomCreateContent.roomType = roomType;
```

Comment:
```suggestion
        MXJSONModelSetString(roomCreateContent.roomType, jsonDictionary[kRoomCreateContentRoomTypeMSC1772JSONKey]);
```

Reply:
I was to prepare the migration between `kRoomCreateContentRoomTypeMSC1772JSONKey` and `kRoomCreateContentRoomTypeJSONKey` once MSC finalized but we can do it late

Reply:
Sure, no problem